src/vm/argdestination.h

```diff
@@ -162,10 +178,33 @@ class ArgDestination
             INDEBUG(remainingBytes -= eightByteSize;)
         }
 
-        _ASSERTE(remainingBytes == 0);        
+        _ASSERTE(remainingBytes == 0);
+
+#endif // #if defined(_TARGET_ARM64_)
     }
 
 #endif //DACCESS_COMPILE
+#endif // (UNIX_AMD64_ABI && FEATURE_UNIX_AMD64_STRUCT_PASSING) || _TARGET_ARM64_
+
+#if defined(UNIX_AMD64_ABI) && defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
+
```

Comment:
shouldn't function  "void ZeroStructInRegisters(int fieldBytes)" also be moved out of ifdef arm64